src/app/App.jsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import { Router, Switch, Route } from 'react-router-dom';
 import { createBrowserHistory } from 'history';
+import styled from 'styled-components';
 
-import { Header } from '../containers/Header';
+import { Header } from '../containers/Header/Header';
```

Comment:
Create an index file with a default export for Header.
And do `import Header from '../containers/Header';`

Reply:
Do it for each container and component. This statement is much shorter and prettier.